core/src/test/kotlin/io/specmatic/conversions/OpenApiSpecificationTest.kt

```diff
@@ -10799,6 +10799,13 @@ paths:
         ))
     }
 
+    @Test
+    fun should_pass_checkSpecValidity_with_valid_openapi_file() {
```

Comment:
Similarly, add another test that checks that checkSpecValidity throws a ContractTestException with the relevant message when an invalid spec is passed.